Doc/library/tkinter.font.rst

```diff
@@ -47,7 +47,16 @@ The different font weights and slants are:
 
    .. method:: actual(option=None, displayof=None)
 
-      Return the attributes of the font.
+      Returns information about the actual attributes that are obtained when
+      the font is used on *displayof*; the values obtained may differ from
+      those given (and retrieved by :meth:`Font.cget`) due to
```

Comment:
"those given" - given where?

I suggest something like "those set in the constructor or via `.config()`".